### ISSUE TITLE
[Screen reader -Howdyai]: Role is incorrectly defined for “Launch Howdy in slack” control in howdy page. 

### ISSUE THREAD

**Issue by v-gasikk** [howdyai/howdy-website#6]
**User Experience**
Users who rely on screen readers will affect to navigate the controls if Role is incorrectly defined for control then the user will be affected so that user will not able to identify the type of control.

**Test Environment**
OS: Win 10 Version 1809 (OS Build 17763.316)
URL: https://howdy.ai/
Browser: Edge
Screen reader: Narrator
 
**Repro Steps**
Turn on the narrator 
Steps to Follow Install Slack:
Step 1: Open the URL: https://howdy.ai/
Step 2: Navigate to "@howdy for Slack" and select it.
Step 3: Navigate to "Create Workspace" and select it.
Step 4: Sign In with valid credentials to open slack account.
Step 5: Now go to Howdy Home page URL: https://howdy.ai/
Step 6: Repeat Step 2 again and select it.
Step 7: Navigate to "Install" button and select it.
Step 8: Select "Authorize" button and select it.
Step 9: Select "Turn your Boot On"
Step 10: Click on "Continue" button.
Step 11: "Howdy Dashboard" page gets opened.
Step 12: Navigate to "Launch Howdy in Slack".
Step 13: Navigate to "Dashboard" tab and select it.
Step 14: "Howdy Dashboard" page gets opened.
Step 15: Navigate to “Launch Howdy in slack” control 
Step 16: Observe the issue.

**Actual Result**
Role is incorrectly defined for “Launch Howdy in slack” control in howdy page. 
Narrator is reading as ““Launch Howdy in slack” button

**Expected Result** 
Role should be correctly defined for “Launch Howdy in slack” control 
Narrator should read as “Launch Howdy in slack” link.

**Mas Reference**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc=%7B248054a6-5e68-4771-9e1e-242fb5025730%7D&cid=3299947d-a121-47be-a3ee-a6e1a76a2609
[MAS4.1.2_Role for slack.zip](https://github.com/howdyai/howdy-website/files/2973557/MAS4.1.2_Role.for.slack.zip)
![MAS4 1 2_Role for Slack](https://user-images.githubusercontent.com/48272005/54471603-9389b500-47e1-11e9-85f5-4e8f9aaa06ca.png)

**Comment by benbrown** [comment]
Is the appropriate fix for this to make this element a LINK instead of a BUTTON?

Or is there an aria-role I can add to address it?

**Comment by awalia13** [comment]
Hi Ben, Please make this element as Link instead of button as it is redirecting to another screen.

**Comment by benbrown** [comment]
OK! this is done.

**Comment by v-gasikk** [comment]
Verified. Issue is Fixed. Hence Closing the bug.
